muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/workspace-management.js

```diff
@@ -108,6 +113,53 @@
     }
   });
   
+  
+  $.widget("custom.workspaceChatSettings", {
+	  options: {
+	      workspaceEntityId: null,
+	  },
+	  _create: function() {
+	    var workspaceEntityId = this.options.workspaceEntityId;
+	    mApi().chat.workspaceChatSettings.read(workspaceEntityId).callback($.proxy(function (err, workspaceChatSettings) {
+	      if (err) { 
+	        $('.notification-queue').notificationQueue('notification', 'error', err);
+          return;
+	      }
+	      
+	      mApi().chat.workspaceChatSettings.read(workspaceEntityId).callback($.proxy(function (err, workspaceChatSettings) {
+	        	
+	        var data = {};
+	        if (workspaceChatSettings == null){
+	        	workspaceChatSettings.status === "DISABLED"
+	          data.disabled_selected = "selected";
+	          this._setStatus("DISABLED");
```

Comment:
I would just leave _setStatus out and save it the first time it is changed, unless there is a need for it (is there?)